### ISSUE TITLE
Add  download manager options to ZeroHello/files menu, and default way download all files in single not multiuser zites

### ISSUE THREAD

**Issue by mx5kevin** [ZeroNetX/ZeroNet#142]
Add  download manager options to ZeroHello/files menu, and default way download all files in single not multiuser zites

The first problem in zites are not possible to easy way download the full content with larger files. Need a simple option when HTML, CSS, pictures are download the web page will now display correctly. In ZeroHello need a button to download all another large files like video files, zip, rar, PDF, etc because without them the site is unenjoyable. And by default, download the full content of the websites the program. Otherwise, the content will be unavailable in real time. Or the content will disappear permanently. We have very few seeders  all users must receive the full content to be available. Right now, the download system is so flawed that the most basic things haven’t worked for years like downloading optional files when the user download a site. And resume complete downloads easy way in multiuser zites. Because file downloads are paused without user interaction default way and not finished.

And default way run at system startup the program to finish easy way the downloads and correcly seed the files. Another problem on zites where have SEED+ button after system startup or program startup the downloads are not resume.

ZeroHello/files menu add pause, resume, delete, +add file link (to download video files, using file links etc from zites, options like in torrent clients and download managers.

Make it possible if a file exit in another zite than merge users.

Remove 10mb limitation in zites and optional files download and have an alert system what show the disk space and if the content larger than the disk space the user get alerted. There is a preset limit in the system that limits the entire network. Show the total filesize on zites before download. HTML, CSS, Picturs and seperated another files like ZIP, RAR, PDF, Video files, same way as torrent clients and download managers.

Larger files get corrupted .mp4 .zip .rar etc. Many times the program tries to download existing files.

**Comment by slrslr** [comment]
> like downloads are paused without user interaction default way and not finished

yes, it is quite important when distributing site with tens of thousands low peer optional files

> Make it possible if a file exist in another zite than merge users.

this is interesting, so far option is to use maybe BTRFS, ZFS and such de-duplication featuring file systems. Also duplicity exist sometimes within same site when it is contributed by many users

> Remove 10mb limitation ... Show the total filesize on zites before download

i had no problem with this, i think it is useful as it is. Maybe having config option to increase default limit. Like 50, 100 MB not small steps 10, 20MB...

**Comment by mx5kevin** [comment]
As a website owner I experience larger files are not available on web pages. As for available content, everything is downloading very slowly in this network, and in torrent network the files are downloading full speed. A larger file takes at least three times as long to download as on a torrent network. The old team had a very bad idea for users to download chunks of different content. So in the end, there was no bandwidth, and all the content was either unavailable for a very long time or not at all. The great advantage of this network would be that everything is available even offline immediately. The other big problem is enough to be placed the website content in the root folder like in pro servers sign and publish. Do not need to edit another files.

**Comment by mx5kevin** [comment]
A very important thing! When a user clicking a file link in a zite always finish the download. If not have seed, restarted the PC, when the user opening a video file finish the download without seed button, and without overcomplicated options always finish it. Especially in zites like ZeroUp, KopyKate, Youtube, IFS and other file sharing zites.

And for zite admininsrators. Detect the zites what files are optional and what not automatic way without forcing the content creators to editing the content.json file. Many website admininstrator are not programmer if the optional parameter not added the larger files are not downloadable.
